CHANGELOG.md

```diff
@@ -37,6 +40,44 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## 4.0.0
+
+#### Added
+- ColumnFactory for providing factory functions instead of requiring as a tuple (column, factory) 
+    - Added by [Gregory Yu](https://github.com/gregyu)
```

Comment:
Seems like it might be redundant given we call out the PR# for each version release in the releases section. In the future if we start having more collaborators and or larger releases that include PRs from multiple feature branches, it would make sense to do.

Reply:
Ah, in the Changelog, that needs to point to the corresponding release section in the CHANGELOG file